sdk/eventhub/event-hubs/src/eventHubConsumerClient.ts

```diff
@@ -35,14 +35,21 @@ const defaultConsumerClientOptions: Required<Pick<
 };
 
 /**
- * The `EventHubConsumerClient` is the main point of interaction for consuming events in Azure Event Hubs service.
+ * The `EventHubConsumerClient` is the main point of interaction for consuming events from an
+ * Event Hub instance.
```

Comment:
Sorry if this was already stated, but what's the reasoning behind the change to:
> Event Hub __instance__

instead of just

> Event Hub

Reply:
either is fine by me. I wanted to avoid the `Event Hubs service` term as the client allows one to connect to a single event hub at a time.

Reply:
"the main point of interaction" is an odd thing to say to my ears. How about: 

> `EventHubConsumerClient`  consumes events from an Azure Event Hub.

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-js/pull/6736/commits/03027f8c2c6138d2b112a489ef2ea785faa54c76.
Changed to

> The `EventHubConsumerClient` class is used to consume events from an Event Hub.